frontend/src/layout/navigation/navigationLogic.ts

```diff
@@ -23,6 +23,9 @@ export const navigationLogic = kea<navigationLogicType>({
     actions: {
         toggleSideBarBase: true,
         toggleSideBarMobile: true,
+        toggleActivationSideBar: true,
+        showActivationSideBar: true,
+        hideActivationSideBar: true,
```

Comment:
It seems a little unnecessary to have a `toggle` plus a `hide` and a `show`. Theoretically we can just do with a `toggle`, yes?

Reply:
There's certain places where I always want to either show or hide, so it's the toggle that's just an extra, I'd just keep them, since it's just kea actions